src/templates/StandardLayout.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled';
+
+import Header from '@/organisms/Header';
+import Footer from '@/organisms/Footer';
+
+const Layout = styled.div`
+  min-height: 100vh;
+
+  display: flex;
+  flex-direction: column;
+`;
+
+const LayoutContent = styled.div`
+  /* TODO: wrap in component selector */
+  flex-grow: 1;
+
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 10%;
+`;
+
+const StyledHeader = styled(Header)`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 10%;
+`;
+
+const StyledFooter = styled(Footer)`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 10%;
```

Comment:
I set up a spacing theme as defined in `Theme.tsx` which are somewhat arbitrary t-shirt sizes that have defined pixel values (you'll note that they are all some unit of 4). As they are part of theme and given with `ThemeProvider` you can use it in any styled component within the app (assuming it is under the `ThemeProvider` parent component) as it will come through 'for free' by way of styled's props -
```
const StyledFoo = styled(foo)`
  padding: 0 ${ props => props.theme.spacing.m };
`
```